### PR TITLE
Configure pipenv for python tests.

### PR DESCRIPTION
**Pull request by arssher**

### CODE REVIEW

**Comment by kelvich** [comment]
LGTM, @funbringer any comment?

File: test_runner/README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Prerequisites:
       `pytest-3`.)
     - Install using something like this:
         - `pip3 install pytest psycopg2` (Debian or Ubuntu)
+    - Or, if you use `pipenv`, with
+	    - `pipenv install`
```
Comment (kelvich) [inline test_runner/README.md:15]: I would better stick to pipenv only -- manual installation is very fragile in python. If we want support installation without pipenv better solution would be to have requirements.txt, which can be fed to pip directly

**Review by funbringer — APPROVED** [review]

File: test_runner/README.md (hunk 1 above)
Comment (arssher) [reply test_runner/README.md]: Agree. I updated readme accordingly and merged.

**Comment by ericseppanen** [comment]
I don't actually know what this does or why it's better-- could someone explain it to me?

Also, should this change the way we build the code in CI?

**Comment by arssher** [comment]
> I don't actually know what this does or why it's better-- could someone explain it to me?
> 
> Also, should this change the way we build the code in CI?

Well, installing python packages system (or user) wide is pain, as dependencies of different projects conflict with each other. Python has virtual environments which solve this problem, and `pipenv` is wrapper around them making the work a bit more comfortable. It appears to me that `pipenv` is the main community accepted way of managing dependencies in Python now. For instance, with this we get list of dependencies (including exact versions known to be working in .Lock file) in a documented format and easy way to install them.

**Pull request merged**